src/OpenSage.DataViewer/UI/Viewers/ConstView.cs

```diff
@@ -50,7 +50,7 @@ public ConstFileEntry(ConstantEntry entry,int index)
             {
                 Index = index.ToString();
 
-                if (entry.Type == ConstantEntryType.Number)
+                if (entry.Type == ConstantEntryType.Register)
```

Comment:
As a style note for the future, I prefer to use braces even for single-line conditional statements. No need to change it for this time though.